snuba/state/rate_limit.py

```diff
@@ -27,6 +31,57 @@
 rds = get_redis_client(RedisClientKey.RATE_LIMITER)
 
 
+def get_rate_limit_config(
+    per_second: tuple[str, float | None], concurrent: tuple[str, int | None]
+) -> tuple[Any, Any]:
```

Comment:
is the typing on this right? Doesn't seem like it should be `Any`

Reply:
This function signature seems to bake in the assumption that every rate limit is both concurrent and per-second. Maybe that's fine but just something I wanted to throw out there as unnecessarily limiting

Reply:
I decided to keep this opinionated for now, since it lets me keep the code cleaner. If we start to encounter edge cases where this needs to be less restrictive we can change it then.